packages/quantic/force-app/main/default/lwc/quanticRecommendationList/templates/grid.html

```diff
@@ -0,0 +1,28 @@
+<template>
+  <template lwc:if={recommendations.length}>
+    <div class="slds-text-heading_medium slds-var-p-bottom_x-small">
```

Comment:
The `slds-text-heading` is used here to make the label look exactly like the mockups.
Also we are adding the heading levels logic in order to improve a11y by having a well structured page, this doesn't mean that we no longer can style the headings.

Also, in the Salesforce context, relying only the heading HTML tags, will always yield the same out put since headings from h1 to h6 are all styles the same way:
<img width="687" alt="headings" src="https://github.com/coveo/ui-kit/assets/86681870/5c8c8fd9-ae62-4e67-8bb6-99b23041bf8b">